### ISSUE TITLE
wrapped in one conda environment

### ISSUE THREAD

**Issue by Thexiyang** [snandiDS/prokseq-v2.0#7]
Thanks for the nice tool! It seems to be not that easily to install. Is it possible to wrap all tools in one conda environment?

**Comment by jacquikeane** [comment]
Yes, this would be really helpful! Making a conda recipe specifying the dependencies would remove the need for editing configs with location of the dependencies and the relevant executables could be copied to the conda bin and and picked up by default when the conda environment is activated.

**Comment by snandiDS** [comment]
We will try to make a conda recipe. Thanks!
However, we recommend Docker (
https://hub.docker.com/repository/docker/snandids/prokseq-v2.0) to overcome
such dependency issues.
